include/seqan3/utility/views/single_pass_input.hpp

```diff
@@ -92,7 +92,7 @@ class single_pass_input_view : public std::ranges::view_interface<single_pass_in
     ~single_pass_input_view() = default;
 
     //!\brief Construction from the underlying view.
-    explicit single_pass_input_view(urng_t _urng) : state_ptr{new state{std::move(_urng)}}
+    explicit constexpr single_pass_input_view(urng_t _urng) : state_ptr{new state{std::move(_urng)}}
```

Comment:
you sure about the `constexpr` for this view?

Reply:
With pmr, the `new` could be constexpr, I think. But the `constexpr` isn't a guarantee anyway